js-apps/meep-frontend/src/js/containers/exec/view-pane.js

```diff
@@ -0,0 +1,382 @@
+        />
+        <ViewSelectFields
+          currentView={this.props.currentView}
+          onSuccess={this.props.onSuccess}
+          onClose={e => this.onViewPaneClose(e)}
+          sandbox={this.props.sandbox}
+          appIds={appIds}
+          ueIds={ueIds}
+          poaIds={poaIds}
+          view1Name={view1Name}
+          view2Name={view2Name}
+          sourceNodeSelectedView1={this.props.sourceNodeSelectedView1}
+          destNodeSelectedView1={this.props.destNodeSelectedView1}
+          sourceNodeSelectedView2={this.props.sourceNodeSelectedView2}
+          destNodeSelectedView2={this.props.destNodeSelectedView2}
+          changeView1={this.changeView1}
+          changeView2={this.changeView2}
+          changeSourceNodeSelectedView1={this.props.changeSourceNodeSelectedView1}
+          changeDestNodeSelectedView1={this.props.changeDestNodeSelectedView1}
+          changeSourceNodeSelectedView2={this.props.changeSourceNodeSelectedView2}
+          changeDestNodeSelectedView2={this.props.changeDestNodeSelectedView2}
+          dashboardViewsList={dashboardViewsList}
+          showApps={this.props.showApps}
+        />
+
+        <div>
+          <Grid style={{ marginTop: 10 }}>
+            <GridInner>
+              <GridCell span={12}>
+                <Button
+                  outlined
+                  style={styles.button}
+                  onClick={e => this.onViewPaneClose(e)}
+                  data-cy={EXEC_BTN_DASHBOARD_BTN_CLOSE}
+                >
+                  Close
+                </Button>
+              </GridCell>
+            </GridInner>
+          </Grid>
+        </div>
+      </div>
+    );
+  }
+}
+
+const styles = {
+  block: {
+    marginBottom: 20
+  },
+  field: {
+    marginBottom: 20
+  },
+  select: {
+    width: '100%'
+  }
+};
+
+const mapStateToProps = state => {
+  return {
+    currentView: state.ui.execCurrentView,
+    selectedScenarioElement: state.exec.selectedScenarioElement,
+    page: state.ui.page,
+    displayedScenario: state.exec.displayedScenario,
+    sourceNodeSelectedView1: state.ui.sourceNodeSelectedView1,
+    destNodeSelectedView1: state.ui.destNodeSelectedView1,
+    sourceNodeSelectedView2: state.ui.sourceNodeSelectedView2,
+    destNodeSelectedView2: state.ui.destNodeSelectedView2,
+    eventCreationMode: state.exec.eventCreationMode,
+    scenarioState: state.exec.state.scenario,
+    view1Name: state.ui.dashboardView1,
+    view2Name: state.ui.dashboardView2,
+    sandboxCfg: state.ui.sandboxCfg,
+    dashboardOptions: state.monitor.dashboardOptions,
+    UEs: execUEs(state),
+    POAs: execPOAs(state),
+    EDGEs: execEdges(state),
+    FOGs: execFogs(state),
+    ZONEs: execZones(state),
+    DNs: execDNs(state),
+    MobTypes: execMobTypes(state),
+    EdgeApps: execEdgeApps(state),
+    FogEdges: execFogEdges(state),
+    table: state.exec.table,
+    networkElements: state.exec.table.entries,
+    showApps: state.ui.execShowApps
```

Comment:
Make sure you are only observing the required state.

Reply:
removed all the extra values that are not needed in both this file and the detailed pane file